analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -170,14 +170,14 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
          * SCM URL still is a prefix of the child POM's SCM URL.
          */
         fun getOriginalScm(mavenProject: MavenProject): Scm? {
-            var scm = mavenProject.scm
+            val scm = mavenProject.scm
             var parent = mavenProject.parent
 
             while (parent != null) {
                 parent.scm?.let { parentScm ->
                     parentScm.connection?.let { parentConnection ->
                         if (parentConnection.isNotBlank() && scm.connection.startsWith(parentConnection)) {
-                            scm = parentScm
+                            scm.connection = parentScm.connection
```

Comment:
Not directly related to this change, but I just noticed that the function docs are confusing because they only talk about the "SCM URL" when actually so far only the connection is compared. After this PR both the URL and connection are processed, could you maybe add a commit to update the documentation?

Reply:
Commit message:
- "child'd" -> "child's"
- no comma in "was, that"

Reply:
There's still one more "child'd".